docs/source/community/content-community.rst

```diff
@@ -2,6 +2,49 @@
 Community Guides
 ================
 
+
+Welcome to the Community Guide for Jupyter. This section try to introduce many
+of the Jupyter-related topics, event, etc, and we welcome any contribution to
+it. As our community is highly dynamic, information there can become out of
+date quickly. Feel free to tell us when this is the case, or if anything is
+missing.
+
+
+Weekly Dev meeting
+------------------
+
+The core developer have weekly dev-meeting to request help, or discuss things
+that happen recently, as well as bootstrap conversation. We try to keep the
+meeting short. These meeting are public and posted on the `IPython channel
+<https://www.youtube.com/channel/UCUuzz1eYiKIzu_Uw1ZQLNoQ>`_ on YouTube.
+
+You will find there two kinds of meetings, the ``IPython/Jupyter``  ones
+– usually at 9am Pacific time on Tuesday – that are broad topic, and the
+``JupyterLab`` ones – usually 9am Pacific Time on Friday –  which are more
+focused on the current evolution of the Jupyter Notebook. These meeting are not
+live broadcast for technical reasons, but we are always looking for a better
```

Comment:
"broadcast live", I would also remove the "for technical reasons" (someone will then open an issue to discuss all the ways to solve the "technical issues", we don't need to get into that :)